src/main/java/com/TableTalk/Enterprise/controllers/TableTalkController.java

```diff
@@ -1,168 +1,119 @@
 package com.TableTalk.Enterprise.controllers;
```

Comment:
In this class I cleaned up imports and changed the way things are passed around.